domains/certificates/.editorconfig

```diff
@@ -0,0 +1,128 @@
+###############################
+# Core EditorConfig Options   #
+###############################
+root=true
```

Comment:
Do we really want this to be root=true? I would be fine with just overwriting the parts of editorconfig that we want.